web/src/features/dataset/components/DatasetTable.tsx

```
@@ -108,7 +108,9 @@ export const DatasetTable = (tabularDataset: DatasetTableProps) => {
                                   <DropdownMenu.Item
                                     key={option}
                                     onClick={() =>
-                                      void handleSetColumnDataPermissionLevel(tabularDataset.columnIds[column]!, option)
+                                      void handleSetColumnDataPermissionLevel(tabularDataset.columnIds[column]!, {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)